Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
```

Comment:
The command for the `dev_server` task is quite long, making it prone to errors and hard to read. Consider breaking this command down into multiple lines for better readability. You may also want to consider extracting certain parameters into environment variables for clarity and reusability.